src/styles/StyleUtils.js

```diff
@@ -1329,6 +1329,35 @@ function getDropDownButtonHeight(buttonSize) {
     };
 }
 
+/**
+ * Returns fitting fontSize and lineHeight values in order to prevent large amounts from being cut off on small screen widths.
+ *
+ * @param {Number} baseFontSize
+ * @param {Number} baseLineHeight
+ * @param {Boolean} isSmallScreenWidth
+ * @param {Number} windowWidth
+ * @param {Number} requestAmount
+ * @returns {Object}
+ */
+function getAmountFontSizeAndLineHeight(baseFontSize, baseLineHeight, isSmallScreenWidth, windowWidth, requestAmount) {
+    let toSubtract = 0;
+
+    if (isSmallScreenWidth) {
+        const widthDifference = variables.mobileResponsiveWidthBreakpoint - windowWidth;
+        if (widthDifference > 350) toSubtract = 2;
+        if (widthDifference > 400) toSubtract = 6;
+        if (widthDifference > 450) toSubtract = 9;
```

Comment:
Maybe a switch range looks cleaner? https://www.digitalocean.com/community/tutorials/how-to-use-the-switch-statement-in-javascript#switch-ranges

Reply:
@pecanoro Replaced with switch range.